Index: frontend/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Logs\r\nlogs\r\n*.log\r\nnpm-debug.log*\r\nyarn-debug.log*\r\nyarn-error.log*\r\npnpm-debug.log*\r\nlerna-debug.log*\r\n\r\nnode_modules\r\ndist\r\ndist-ssr\r\n*.local\r\n\r\n# Editor directories and files\r\n.vscode/*\r\n!.vscode/extensions.json\r\n.idea\r\n.DS_Store\r\n*.suo\r\n*.ntvs*\r\n*.njsproj\r\n*.sln\r\n*.sw?\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/frontend/.gitignore b/frontend/.gitignore
--- a/frontend/.gitignore	(revision 9640550aad9448e12cebeea599011aadd72e0063)
+++ b/frontend/.gitignore	(date 1719998554249)
@@ -22,3 +22,5 @@
 *.njsproj
 *.sln
 *.sw?
+
+.env
diff --git a/backend/README.md b/README.md
rename from backend/README.md
rename to README.md
